src/templates/list-table-block/index.vue

```diff
@@ -293,6 +306,9 @@
   ])
 
   const props = defineProps({
+    hiddenHeader: {
```

Comment:
maybe is better to use showHeader and use the default value as true, then in the template we can check v-if="showHeader", this will be also the default behavior, and when is required to hide the Header the developer will pass showHeader="false"

Reply:
Thank you for the suggestion! However, I believe it's more appropriate to keep the prop as hiddenHeader. In Vue, boolean props are false by default, and when passed as <element hiddenHeader>, they are interpreted as true. This approach more intuitively reflects the intention of hiding the header.

By keeping the prop as hiddenHeader, we maintain a clearer and more straightforward logic: if the goal is to hide the header, you simply pass this prop; otherwise, the header will be displayed by default.